villain/ast.rkt

```diff
@@ -12,8 +12,8 @@
 ;;           | (Int Integer)
 ;;           | (Bool Boolean)
 ;;           | (Char Character)
-;;           | (String)         
-;;           | (Symbol Symbol)
+;;           | (String s)         
```

Comment:
`s` should be `String` here and `Symbol` below.  (It's a little weird since the name of the constructor and the type of the argument are the same, but saying `s` doesn't make sense.)